docs/content/integrations/airflow/from-airflow-to-dagster.mdx

```diff
@@ -103,13 +105,15 @@ this yields us the following graph of computations
   />
 </center>
 
-##### Op Level Retries
+### Op-level retries
 
-In the tutorial DAG, the `t2` operator allowed for 3 retries. To configure the same behavior in Dagster, you can use [op level retry policies](/concepts/ops-jobs-graphs/op-retries)
+In the tutorial DAG, the `t2` operator allowed for 3 retries. To configure the same behavior in Dagster, you can use [op-level retry policies](/concepts/ops-jobs-graphs/op-retries)
```

Comment:
```suggestion
In the tutorial DAG, the `t2` operator allowed for three retries. To configure the same behavior in Dagster, you can use [op-level retry policies](/concepts/ops-jobs-graphs/op-retries).
```

Missing a period here, and numbers under 10 should be spelled out unless you're referring to a literal value. Ex: "The value of some field is `3`"